tests/allocator-test.cc

```diff
@@ -471,21 +499,19 @@ namespace
 		     ptr2);
 		state = 2;
 		// Yield to allow the hazards to be dropped.
-		Timeout t{1};
-		thread_sleep(&t);
+		TEST(sleep(1) >= 0, "Failed to yield to drop hazards");
 		// Try a double free.  This may logically succeed, but should not affect
 		// our quota.
-		heap_free(SECOND_HEAP, ptr);
+		(void)heap_free(SECOND_HEAP, ptr);
```

Comment:
Why doesn't this one test the return value of `heap_free`?

Reply:
Hm, I don't know why I just dropped that one!  Fixed.